content/en/docs/concepts/scheduling-eviction/assign-pod-node.md

```diff
@@ -441,7 +448,7 @@ The design documents for
 [node affinity](https://git.k8s.io/community/contributors/design-proposals/scheduling/nodeaffinity.md)
 and for [inter-pod affinity/anti-affinity](https://git.k8s.io/community/contributors/design-proposals/scheduling/podaffinity.md) contain extra background information about these features.
 
-Once a Pod is assigned to a Node, the kubelet runs the Pod and allocates node-local resources.
+Once a Pod is  successfully assigned to a Node, the kubelet runs the Pod and allocates node-local resources.
```

Comment:
More than single whitespace before `successfully`. Also not sure about the need for `successfully` since it is understood